src/client/components/toggleZone/toggleZone.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import toJson from 'enzyme-to-json';
+
+import ToggleZone from './toggleZone';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
Delete this string. Enzyme configure already configured in settings.

Reply:
done